template_content/.algokit.toml.jinja

```diff
@@ -0,0 +1,21 @@
+[algokit]
+min_version = "v1.1.0-beta.4"
+
+[deploy]
+{%- if deployment_language == 'python' %}
+command = "poetry run python -m smart_contracts deploy"
+{%- elif deployment_language == 'typescript' %}
+command = "npm run deploy:ci"
+{%- endif %}
+environment_secrets = []
+
+[deploy.localnet]
+environment_secrets = []
+
+[deploy.testnet]
+environment_secrets = [
```

Comment:
I think this should be back under the parent `[deploy]` section as that is a better indication that these secrets are used in all environments *except* localnet. Having it just in testnet implies that these secrets are only for use in testnet